tests/integration/devfile/cmd_devfile_create_test.go

```diff
@@ -184,4 +184,36 @@ var _ = Describe("odo devfile create command tests", func() {
 	//		helper.DeleteDir(contextDevfile)
 	//	})
 	//})
+
+	// Context("When executing odo create with devfile component, --downloadSource flag and sparseContextDir has a valid value", func() {
```

Comment:
👍 Thanks for writing the tests even if they can't be enabled just yet.

A couple small modifications I'd recommend:
- Add the `--devfile` flag to the `odo create` commands, this way we're not relying on interactive mode (and thus don't need to wait for the integration tests to support it). Then we only need to wait for https://github.com/openshift/odo/issues/2862 to be delivered.